java/src/Ice/src/main/java/com/zeroc/Ice/_ObjectPrxI.java

```diff
@@ -395,22 +395,22 @@ public ObjectPrx ice_adapterId(String newAdapterId) {
       newAdapterId = "";
     }
 
-    if (newAdapterId.equals(_getReference().getAdapterId())) {
+    if (newAdapterId.equals(_reference.getAdapterId())) {
```

Comment:
I don't understandand why you replaced calls to _getReference() by _reference only in some method of this file. Should it be a global replace instead?